tracer/test/test-applications/integrations/Samples.Yarp.DistributedTracing/Samples.Yarp.DistributedTracing.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">1.1.0</ApiVersion>
```

Comment:
We don't do the multi-version test against this version on Linux, but we test against it in Windows it seems - any specific reason for that setup?

Reply:
My intent was to multi-version test this on Linux, did I author something incorrectly?

Reply:
I see the Linux integration tests building multiple versions of `Samples.Yarp.DistributedTracing` so I think it's working correctly. Let me know if you see something that indicates Linux isn't working but I think I'll continue with the PR with the assumption that Linux multi-version testing is OK

Reply:
Sorry, what I meant was that we had an explicit lower bound on the version we test of 1.1.2 in the linux multi-version tests, yet here we're setting it to 1.1.0 (which would be used on Windows). I think missing those earlier versions was just an oversight and is now fixed IIUC 🙂